src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -257,6 +257,9 @@ void InitCapabilities() {
       std::string("menuIcon"), hmi_apis::Common_ImageFieldName::menuIcon));
   image_field_name_enum.insert(std::make_pair(
       std::string("cmdIcon"), hmi_apis::Common_ImageFieldName::cmdIcon));
+  image_field_name_enum.insert(
```

Comment:
@VVeremjova Why did you add this to hmi_capabilities?. This map using for convertation string parameter from hmi_capabilities.json to enum. I didn't see that you add some parameters to hmi_capabilities.json file.